edb/server/compiler/errormech.py

```diff
@@ -453,9 +453,15 @@ def _static_interpret_invalid_param_value(
     err_details: ErrorDetails,
     from_graphql: bool = False,
 ):
+    error_message_context = ''
+    if err_details.detail_json:
+        error_message_context = (
+            err_details.detail_json.get('error_message_context', '')
+        )
+
```

Comment:
The big question for me with this PR is:
Is it better to have the context prepended to the actual message, or would it be better to put it in the `details` field?

My inclination had been `details`, but this looks pretty nice.

Reply:
I guess either works, but it seemed to me that details were often more about providing information from beyond the immediate context. (eg. stuff about the inheritance of the types involved in an error)